config/samples/numaplane.numaproj.io_v1alpha1_monovertexrollout.yaml

```diff
@@ -41,8 +41,47 @@ spec:
             image: quay.io/numaio/numaflow-go/source-simple-source:stable
         transformer:
           container:
-            image: docker.intuit.com/quay-rmt/numaio/numaflow-rs/source-transformer-now:stable
+            image: quay.io/numaio/numaflow-rs/source-transformer-now:stable
```

Comment:
that shouldn't have been going to the Intuit proxy, even though it worked for us